bgpd/bgp_mplsvpn.c

```diff
@@ -1715,9 +1717,11 @@ void vpn_leak_to_vrf_update(struct bgp *bgp_vpn,	    /* from */
 
 		if (!path_vpn->extra
 		    || path_vpn->extra->bgp_orig != bgp) { /* no loop */
-			vpn_leak_to_vrf_update_onevrf(bgp, bgp_vpn, path_vpn);
+			leak_success |= vpn_leak_to_vrf_update_onevrf(
```

Comment:
`|=` => `=`?
```c
leak_success |= vpn_leak_to_vrf_update_onevrf(
```

Reply:
I mean to use '|' to return true if at least one of the imported entries is true.

1|0 = 1

Reply:
This is a bit unusual pattern (at least it makes me confusing when I read it).

Reply:
do you have an other pattern to recommend ?
I think that one is optimized.